gems/smithy/spec/interfaces/client/types_spec.rb

```diff
@@ -4,27 +4,46 @@
   %i[schema client].each do |plan_type|
     context "#{plan_type} generator" do
       before(:all) do
-        @tmpdir = SpecHelper.generate(['Weather'], plan_type)
+        @tmpdir = SpecHelper.generate(['AllShapes'], plan_type)
```

Comment:
This is fine in general. Weather was useful to get started. We have shapes and all_shapes. I eventually want one of them.. so no action needed